src/core/PWSFilters.cpp

```diff
@@ -769,6 +775,7 @@ stringT PWSFilters::GetFilterDescription(const st_FilterRow &st_fldata)
         break;
       }
       // Note: purpose drop through to standard 'string' processing
+      [[fallthrough]];
```

Comment:
Is this project already supporting C++17 features?

Reply:
gcc 9 process them even in c++11 mode, but now I've commented them (see detail in the first comment)